src/renderer/containers/Console/AddConnectionForm.tsx

```diff
@@ -108,50 +114,79 @@ export const AddConnectionForm: React.FC<AddConnectionFormProps> = ({ defaultVal
-              css={css`
-                display: grid;
-                grid-gap: 10px;
-                grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
-                margin-bottom: 10px;
-              `}
-            >
-              <TextField
-                label="OBS Websocket IP:Port"
-                value={obsIP ?? ""}
-                required={showAdvanced}
-                onChange={(e) => setValue("obsIP", e.target.value)}
+              description={
+                <span
+                  css={css`
+                    a {
+                      text-decoration: underline;
+                    }
+                  `}
+                >
+                  Enables automatic hiding and showing of an OBS source (e.g. your Dolphin capture) when the game is
+                  active. Requires <A href="https://github.com/Palakis/obs-websocket">OBS Websocket Plugin</A>.
+                </span>
+              }
+            />
+            <section>
+              <Collapse in={showAutoswitcher}>
+                <div
+                  css={css`
+                    display: grid;
+                    grid-gap: 10px;
+                    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+                    margin: 10px 0px 10px;
+                  `}
+                >
+                  <TextField
+                    label="OBS Websocket IP:Port"
+                    value={obsIP ?? ""}
+                    required={showAutoswitcher}
+                    onChange={(e) => setValue("obsIP", e.target.value)}
+                  />
+                  <TextField
+                    label="OBS Password"
+                    value={obsPassword ?? ""}
+                    onChange={(e) => setValue("obsPassword", e.target.value)}
+                    type="password"
+                  />
+                </div>
+                <TextField
+                  label="OBS Source Name"
+                  value={obsSourceName ?? ""}
+                  required={showAutoswitcher}
+                  onChange={(e) => setValue("obsSourceName", e.target.value)}
+                />
+              </Collapse>
+            </section>
+            <section>
+              <Toggle
+                value={Boolean(enableRelay)}
+                onChange={(checked) => setValue("enableRelay", checked)}
+                label="Console Relay"
+                description="The relay allows external programs (e.g. stream layouts) to tap into the raw Slippi data stream without affecting mirroring.
+                This connection's relay port will be shown on the console card after you have saved and is activated once you select connect."
               />
-              <TextField
-                label="OBS Password"
-                value={obsPassword ?? ""}
-                onChange={(e) => setValue("obsPassword", e.target.value)}
-                type="password"
+            </section>
+            <section>
+              <Toggle
```

Comment:
I don't really think this needs to be hidden behind a toggle. It should probably just a be titled "Connection Port" and have it be a text field which defaults to the Wii port.